src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -249,6 +258,26 @@ private CallbackResponse prePopulateOrderDetailsPages(CallbackParams callbackPar
 
         updatedData.disposalHearingFinalDisposalHearing(tempDisposalHearingFinalDisposalHearing).build();
 
+        // updated Hearing time field copy of the above field, leaving above field in as requested to not break
+        // existing cases
+        DisposalHearingHearingTime tempDisposalHearingHearingTime =
```

Comment:
would this change need to go behind a v1 callback since the field disposalHearingHearingTime is in an H&L-nonprod file?

Reply:
checked this and youre correct, ive moved under a feature toggle